### PR TITLE
Order notes addition

### PR DESCRIPTION
**Pull request by yashgit891**
### - Link added should be a valid URL and accessible throughout the org.
   ### - If the branch name contains hotfix / revert by default the BVT workflow check will pass.

| Test Case Document URL                        |
|-----------------------------------------------|
| Please paste test case document link here.... |

### CODE REVIEW

File: Controller/Payment/Callback.php [hunk 1]
```diff
@@ -204,6 +209,29 @@ public function execute()
                     ->load($order->getQuoteId());
                 $quote->setIsActive(false)
                     ->save();
+                
+                $amountPaid = number_format($rzpPayment['amount'] / 100, 2, ".", "");
+        
+                if ($rzpPayment['status'] === static::AUTHORIZED)
```
Comment (ramth05) [inline Controller/Payment/Callback.php]: @yashgit891 , why two blocks , use variables. As message looks similar.

File: Controller/Payment/Callback.php [hunk 2]
```diff
@@ -173,6 +176,8 @@ public function execute()
                 $payment = $order->getPayment();
                 $paymentId = $params['razorpay_payment_id'];
 
+                $rzpPayment = $this->rzp->request->request('GET', 'payments/'.$paymentId);
```
Comment (ramth05) [inline Controller/Payment/Callback.php:179]: @yashgit891 , why to poll payments API ? can't we use webhook to update this notes?

Comment (yashgit891) [reply Controller/Payment/Callback.php:179]: @ramth05 then we will have to accommodate this in the update order cron, else again latency issue will start.

Comment (ramth05) [reply Controller/Payment/Callback.php:179]: yes, that works. In place of adding latency with callback.

**Review by ramth05 — APPROVED** [review]
LGTM

**Pull request merged**